README.md

```diff
@@ -20,7 +20,7 @@ limitations under the License.
 [![PyPI version](https://badge.fury.io/py/elyra.svg)](https://badge.fury.io/py/elyra)
 [![Anaconda-Server Badge](https://anaconda.org/conda-forge/elyra/badges/version.svg)](https://anaconda.org/conda-forge/elyra)
 [![Downloads](https://pepy.tech/badge/elyra)](https://pepy.tech/project/elyra)
-[![Documentation Status](https://readthedocs.org/projects/elyra/badge/?version=latest)](https://elyra.readthedocs.io/en/latest/?badge=latest)
+[![Documentation Status](https://readthedocs.org/projects/elyra/badge/?version=stable)](https://elyra.readthedocs.io/en/stable/?badge=stable)
```

Comment:
This should not be changed permanently. It would be ok to change in the release script.

Reply:
Wy not? [New] users are likely going to access the documentation from the readme if they don't have Elyra installed yet and they should see by default  the documentation for code that is published, not being worked on.

Reply:
The badge describes the build status for documentation. When looking into master README it should describe the status of the master documentation build. 

I believe what we agreed on was to change the "right side documentation link" to point to stable. 

![image](https://user-images.githubusercontent.com/382917/115442953-79dca800-a1c7-11eb-95e8-373b4f05d15f.png)

Reply:
Ah, I missed that part. Do note that the JupyterLab badge in their repo points to the stable version!